acceptance/tests/environment/negative/agent_run_should_fail_if_env_unreadable.rb

```diff
@@ -0,0 +1,58 @@
+test_name "C97899 - Agent run should fail if environment is unreadable" do
+  skip_test 'requires a master for managing the environment' if hosts_with_role(hosts, 'master').length == 0
+
+  testdir = ''
+  env_path = ''
+  test_env = ''
```

Comment:
nit, it's not necessary to set these to `''` since we set them on lines 9-11 below, and there isn't conditional logic that could cause them to not be set. Not blocking this PR, just general comment.